src/AppBundle/Entity/ProcurationRequest.php

```diff
@@ -320,7 +335,7 @@ private static function createPhoneNumber(int $countryCode = 33, string $number
      *
      * @param ExecutionContextInterface $context
      */
-    public function validateElectionsChosen(ExecutionContextInterface $context)
+    public function validateChosenElections(ExecutionContextInterface $context): void
```

Comment:
@hhamon this required change was not related to this PR, or should we also change the constants name to `..._REASON` :trollface:?